<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVSimulationSettingsView</class>
 <widget class="QWidget" name="SVSimulationSettingsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="minimumSize">
      <size>
       <width>220</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page_0"/>
     <widget class="QWidget" name="page1"/>
     <widget class="QWidget" name="pageSimulation">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTabWidget" name="tabWidgetSimulation">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="tabPosition">
          <enum>QTabWidget::North</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabSimulationStart">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <attribute name="title">
           <string>Start Simulation</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tabSimulationOutputs">
          <attribute name="title">
           <string>Outputs</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tabSimulationModelOptions">
          <attribute name="title">
           <string>Model settings</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tabSimulationPerformanceOptions">
          <attribute name="title">
           <string>Performance options</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tabNetworkSettings">
          <attribute name="title">
           <string>Network settings</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page3"/>
     <widget class="QWidget" name="page_4"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
